packages/nextjs/components/Logo.tsx

```diff
@@ -8,23 +8,30 @@ type Props = React.ComponentProps<"svg"> & {
 /**
  * Lisk logo.
  */
-export const Logo: React.FC<Props> = ({ size = 32, ...props }) => {
+export const Logo: React.FC<Props> = ({}) => {
```

Comment:
The Lisk logo used here is not the correct one. It should be similar to the favicon, just like in the website and Portal.

Reply:
If no props are passed in, then the empty object should either be removed or the size and rest props should still be passed. That way, the width and height of the SVG can always be dependent on the size.

Reply:
Updated! 👍🏼